AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Service/PinpointClientProtocol.swift

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import AWSPinpoint
+import Foundation
+
+public protocol PinpointClientProtocol {
```

Comment:
Oh... if the SDK removed all the client protocols, then unfortunately we'll need to add **all** of the client methods to the ones we provide.
I guess it applies to all the other categories as well, but for Analytics and PushNotifications we return this type in the `getEscapeHatch()` method, which is meant to allow customers to interact with all the supported APIs, not just the ones we use.

I remember when I decided that returning the protocol instead of the client was a "more correct" idea... 🙈 🤡